Add keyboard shortcut

Vbialk opened the pull request:
A keyboard shortcut is added to run the delete-all action using the key combination Ctrl+Alt+D.

Ein Tastatur Shortcut wird eingebaut, mit dem die delete-all Aktion über die Tastenkombination Strg+Alt+D ausgeführt werden kann.

Vbialk committed:
Add keyboard shortcut

A keyboard shortcut is added to run the delete-all action using the key combination Ctrl+Alt+D.

Ein Tastatur Shortcut wird eingebaut, mit dem die delete-all Aktion über die Tastenkombination Strg+Alt+D ausgeführt werden kann.

pathmapper reviewed (commented):
Nice idea adding a shortcut 👍

Some issues with this implementation:

- after installing the plugin, there's no entry in the shortcuts manager
- adds also the shortcut when there's already a shortcut with this key sequence, leading to disfunctional shortcuts
- when setting a custom key sequence, after deactivating/activating the plugin, the tooltip shows a different key sequence compared to the sequence in the shortcuts manager

Vbialk committed:
fix shortcut

The shortcut is now implemented with a MainWindowAction (default shortcut) and the QgsShortcutsManager. If the shortcut is set and no other shortcut is assigned to the combination "Ctrl+Alt+D", the default shortcut is assigned. @pathmapper this should realize your suggetions, please test it :-)

pathmapper commented:
@Vbialk thanks for the update, 

> adds also the shortcut when there's already a shortcut with this key sequence, leading to disfunctional shortcuts

is still an issue.
